language/move-lang/src/cfgir/absint.rs

```diff
@@ -153,12 +153,12 @@ pub trait AbstractInterpreter: TransferFunctions {
 
     fn execute_block(
         &mut self,
-        cfg: &BlockCFG,
+        cfg: &mut dyn CFG,
```

Comment:
Why does this need to be mutable?

Reply:
I had originally assumed I would be using the abstract interpreter frame work to insert the releases. But I think that is not true

Reply:
Yeah, mutating code while trying to compute a fixed point over it sounds ill-advised...  A postprocessing step would probably be cleaner.